lib/requirements.txt

```diff
@@ -12,4 +12,21 @@ joblib~=1.0.1  # BSD
 pandas~=1.1.5  # BSD
 six~=1.15.0  # MIT
 minio~=7.0.3  # Apache-2.0
-uvicorn~=0.14.0  # BSD
\ No newline at end of file
+uvicorn~=0.14.0  # BSD

```

Comment:
this seems scary, since here add all dependences for all cases.

Reply:
updated